repository.go

```diff
@@ -331,7 +343,7 @@ func (r *Repository) buildRepositoryBody(ro *RepositoryOptions) string {
 	//		body["name"] = ro.Name
 	//}
 	if ro.IsPrivate != "" {
-		body["is_private"] = ro.IsPrivate
+		body["is_private"] = strings.ToLower(strings.TrimSpace(ro.IsPrivate)) != "false"
```

Comment:
I thought the below was the right one, and do you think?
```suggestion
		body["is_private"] = strings.ToLower(strings.TrimSpace(ro.IsPrivate)) 
```

Reply:
@ktrysmt Bitbucket expects a boolean value in the payload for the `is_private` field. If you try and submit a request with the previous code you get the error:

```
{"type": "error", "error": {"fields": {"is_private": "expected bool"}, "message": "Bad request"}}
```

The new condition of `!=` makes the value a real boolean, which is what Bitbucket expects.

Reply:
I see.
I understood that `is_private: boolean` is detected at the site.